lte/gateway/c/core/oai/include/map.h

```diff
@@ -179,6 +179,29 @@ struct map_s {
     }
   }
 
+  /***************************************************************************
```

Comment:
can we add a TODO on this file on line 150 to mark that `std::unordered_map` will be changed to `google::protobuf::Map` interface?

Reply:
addressed review comment